modules/openapi-generator/src/main/resources/Java/libraries/native/ApiClient.mustache

```diff
@@ -199,11 +199,11 @@ public class ApiClient {
     return mapper;
   }
 
-  protected String getDefaultBaseUri() {
+  private String getDefaultBaseUri() {
```

Comment:
generally i'm ok with the PR but if later users have a use case that need `protected` method here for example, we will need to provide a way to fallback.